src/redux/categorySlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  item: 'undercontruction',
+};
+
+export const categorySlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    selectCategory: (state, action) => {
+            state.item = action.payload; // eslint-disable-line
+    },
+  },
+});
+export const { selectCategory } = categorySlice.actions;
```

Comment:
- [ ] in this file you should also Define CHECK_STATUS as action type

Reply:
thank you for your time but we are also allowed to use the redux tool kit because
Redux Toolkit simplifies store setup down to a single clear function call, while retaining the ability to fully configure the store's options if you need to
Redux Toolkit eliminates accidental mutations, which have always been the #1 cause of Redux bugs
Redux Toolkit eliminates the need to write any action creators or action types by hand
Redux Toolkit eliminates the need to write manual and error-prone immutable update logic
Redux Toolkit makes it easy to write a Redux feature's code in one file, instead of spreading it across multiple separate files
Redux Toolkit offers excellent TS support, with APIs that are designed to give you excellent type safety and minimize the number of types you have to define in your code
RTK Query can eliminate the need to write any thunks, reducers, action creators, or effect hooks to manage fetching data and tracking loading state

Reply:
Hi @HeDevedUp,
My apologies I wasn't aware of this, please request for another review.

Reply:
no problem sir thank you for your time